languages/ocaml.scm

```diff
@@ -473,6 +473,20 @@
   (_)
 )
 
+; Keep spacing between pointy brackets when used in object types.
+;
+; This is syntactically correct:
+;   let obj_id (obj : < .. >) = obj
+; This is not:
+;   let obj_id (obj : <..>) = obj
+(object_type
+  "<" @append_space
+  .
+  (_)*
+  .
```

Comment:
Aren't these lines redundant?

Reply:
This is redundant indeed. Fixed in latest commit